tests/behavior_test.cpp

```diff
@@ -155,6 +155,7 @@ TEST_CASE( "behavior_tree", "[behavior]" )
 TEST_CASE( "check_npc_behavior_tree", "[npc][behavior]" )
 {
     clear_map();
+    calendar::turn = calendar::start_of_cataclysm;
```

Comment:
Why this line? this seems unrelated to the rest of the PR

Reply:
Warmth from sun is now affected by weather and time of day in wider range of situations.

That is there to set time to middle of night to prevent warmth from sun from messing up the results.